node/src/test_utils/mod.rs

```diff
@@ -529,6 +535,35 @@ pub struct TestRawTransaction {
     pub data: Vec<u8>,
 }
 
+pub fn make_node_base_dir_and_return_its_absolute_and_relative_path_to_os_home_dir(
```

Comment:
You mentioned that "why it still exists?". I think you'd like to remove it and change the code inside tests if it's used there. Good Luck :+1: